examples/html/index.html

```diff
@@ -2,22 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>HTML Example</title>
-    <link rel="stylesheet" href="style.css" />
   </head>
```

Comment:
Why remove these

Reply:
All these removals were intentional to simplify the examples. I had same idea about wagmi section examples, but it is unnecessary chore for us to keep them updated as opposed to linking to wagmi. Instead, I am planning on full blown dapp example in our `web-examples` repo that showcases them at some point. But for examples within this repo, I think simplest possible ones do the trick.